content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -444,6 +444,39 @@ securityContext:
 To assign SELinux labels, the SELinux security module must be loaded on the host operating system.
 {{< /note >}}
 
+### Speeding up SELinux volume relabeling
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+By default, the contrainer runtime recursively assigns SELinux label to all
+files on all Pod volumes. To speed up this process, Kubernetes can change the
+SELinux label of a volume instantly by using a mount option
+`-o context=<label>`.
+
+To benefit from this speedup, all these conditions must be met:
+
+* Alpha feature gates `ReadWriteOncePod` and `SELinuxMountReadWriteOncePod` must
+  be enabled.
+* Pod must use PersistentVolumeClaim with `accessModes: ["ReadWriteOncePod"]`.
+* Pod (or all its Containers that use the PersistentVolumeClaim) must
+  have `seLinuxOptions` set.
+* Corresponding PersistentVolume must be either a CSI volume or an in-tree iSCSI
+  volume.
+  * For the CSI volume, the corresponding CSI driver must announce that it
+    supports mounting with `-o context` by setting `spec.seLinuxMount: true` in
+    its CSIDriver instance.
+
+All other volumes will be relabeled by the container runtime by recursively
+changing SELinux label of all files on the volumes.
+
+{{< note >}}
+In Kubernetes 1.25, kubelet loses track of volume labels after restart. In
+other words, kubelet may refuse to start Pods with errors like "conflicting
+SELinux labels of volume", while there are no conflicting labels in Pods. Make sure
+nodes are fully drained before restarting kubelet. This will be addressed in a
+future Kubernetes release before graduating the feature to Beta.
```

Comment:
We [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) unless these are firm commitments that SIG Architecture would stand behind.

If not fixing that bug for beta would be a release blocker, **and this is clearly defined in the KEP**, then this statement is OK to include. Even then, I'd be minded to omit it. Folks are reading docs for a Kubernetes release that has happened and usually don't need to find out how to operate Kubernetes releases we haven't yet published.

Also also: consider linking to https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/

Reply:
Ack, added link to node draining + removed "This will be addressed in a future Kubernetes release before graduating the feature to Beta.".